cost-analyzer/values.yaml

```diff
@@ -731,6 +731,8 @@ service:
   # nodePort:
   labels: {}
   annotations: {}
+  loadBalancerIP: ""
+  loadBalancerSourceRanges: []
```

Comment:
comment here explaining the use of this param?

Reply:
We want to allow only some IP's to access Kubecost

Reply:
I would prefer these be commented out by default since they aren't applicable when in ClusterIP mode.

Reply:
That's true. That's why,  I have a condition like below:
```
{{- if (eq .Values.service.type "LoadBalancer") }}
  {{- if .Values.service.loadBalancerSourceRanges }}
  loadBalancerSourceRanges:
{{ toYaml .Values.service.loadBalancerSourceRanges | indent 4 }}
  {{- end -}
```}

Reply:
Yes, I understand, but it's cleaner if non-default and especially corner case values be commented out by default. This is a practice we try and follow in this chart. It's also a common practice.

Reply:
Got it. That means when I need it, I can uncomment and use it. Will make the change

Reply:
Updated the same